server/src/ClientToDBAPI/CountiesRoute/mainRoute.ts

```diff
@@ -1,54 +1,51 @@
 import { Router, Request, Response } from 'express';
 
-import logger from '../../Logger/Logger';
-import { graphqlRequest } from '../../GraphqlHTTPRequest';
 import { Severity } from '../../Models/Logger/types';
-import { GET_COUNTY_DISPLAY_NAME_BY_USER, GET_ALL_COUNTIES } from '../../DBService/Counties/Query';
+import { errorStatusCode, graphqlRequest } from '../../GraphqlHTTPRequest';
 import GetAllCountiesResponse from '../../Models/User/GetAllCountiesResponse';
+import { GET_COUNTY_DISPLAY_NAME_BY_USER, GET_ALL_COUNTIES } from '../../DBService/Counties/Query';
+import logger, { invalidDBResponseLog, launchingDBRequestLog, validDBResponseLog } from '../../Logger/Logger';
 
 const countiesRoute = Router();
-const errorStatusCode = 500;
 
 countiesRoute.get('', (request: Request, response: Response) => {
     const getCountiesLogger = logger.setup({
-        workflow: 'All Counties Query',
+        workflow: 'query all counties',
     });
     graphqlRequest(GET_ALL_COUNTIES, response.locals)
         .then((result: GetAllCountiesResponse) => {
-            if (result?.data?.allCounties?.nodes) {
-                getCountiesLogger.info('Queried all counties successfully', Severity.LOW);
-                let counties = result.data.allCounties.nodes.map((county: any) => ({
-                    id: county.id,
-                    displayName: county.displayName,
-                    district: county.districtByDistrictId.displayName
-                }));
-                response.send(counties);
-            } else {
-                getCountiesLogger.error(`couldnt query all counties due to ${result.errors[0].message}`, Severity.CRITICAL);
-                response.status(errorStatusCode).send(`Couldn't query all counties`);
-            }
+            getCountiesLogger.info(validDBResponseLog, Severity.LOW);
+            const counties = result.data.allCounties.nodes.map((county: any) => ({
```

Comment:
isn't it better to do something like { ...county , district : county.districtByDistrictId.displayName} instead?